rustls/src/server/tls13.rs

```diff
@@ -604,7 +604,8 @@ mod client_hello {
         common.send_msg(m, false);
     }
 
-    #[allow(clippy::needless_pass_by_ref_mut)] // cx only mutated if cfg(feature = "quic")
+    #[allow(unknown_lints)]
+    #[allow(clippy::needless_pass_by_ref_mut)]
```

Comment:
Concretely, this seems to work for me:

```suggestion
    #[cfg_attr(not(feature = "quic"), allow(clippy::needless_pass_by_ref_mut))]
```